Dockerfile

```diff
@@ -1,31 +1,30 @@
 FROM ubuntu:latest
-LABEL version="1.0" maintainer="John Stucklen <stuckj@gmail.com>"
+LABEL version="1.1" maintainer="John Stucklen <stuckj@gmail.com>"
 
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
 ENV SUBSONIC_UID 1000
 ENV SUBSONIC_GID 1000
 
-RUN apt update \
-  && apt upgrade -y \
-  && apt install -y locales ffmpeg openjdk-8-jre-headless nano flac lame mikmod timidity wget \
-  && apt clean \
+RUN export DEBIAN_FRONTEND=noninteractive \
+  && apt-get update \
```

Comment:
Ah, I never knew what the difference was between `apt` and `apt-get`. Good to know.